akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -80,7 +80,10 @@ object EntityStreamException {
  * The limit can also be configured in code, by calling [[HttpEntity#withSizeLimit]]
  * on the entity before materializing its `dataBytes` stream.
  */
-case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
+final case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
```

Comment:
Technically, constructing is fine, extending is not :)

On Mon, Mar 14, 2016 at 10:40 AM, Konrad Malawski <notifications@github.com>
wrote:

> In akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala
> <https://github.com/akka/akka/pull/20014#discussion_r55974757>:
>
> > @@ -80,7 +80,10 @@ object EntityStreamException {
> >   * The limit can also be configured in code, by calling [[HttpEntity#withSizeLimit]]
> >   * on the entity before materializing its `dataBytes` stream.
> >   */
> > -case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
> > +final case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
>
> Yes and no. Technically it is not, in practice no one should have been
> constructing this exception in the first place other than us.
>
> cc @patriknw <https://github.com/patriknw>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/20014/files#r55974757>.
>



-- 
Cheers,
√